docs/rootfs-and-kernel-setup.md

```diff
@@ -74,14 +74,13 @@ on an aarch64 machine.
 
 ## Creating a rootfs Image
 
-A rootfs image is just a file system image, that hosts at least an init
-system. For instance, our getting started guide uses an EXT4 FS image with
-OpenRC as an init system. Note that, whichever file system you choose to use,
```

Comment:
We just use the default of the rootfs. The new Ubuntu images use systemd. Not sure what the old Ubuntu images used, but  the Alpine initrd images used OpenRC.

I removed references to it since for us it doesn't really matter what init system is used. The customer can use whatever makes sense.